### PR TITLE
T23: Voting Contest

### PR DESCRIPTION
**Pull request by Ashish02112001**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+      #region Methods ---------------------------------------------
+      /// <summary>Gets the votes and gives the winner</summary>
+      /// <param name="args">arguments</param>
       static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+         string S;
+         for (; ; ) {
+            Console.Write ("Enter your vote for contestants(A to Z): ");
+            S = Console.ReadLine () ?? "";
+            if (S.All (char.IsAsciiLetter) && S.Length > 0) {
```
Comment (asfar-nazar) [inline Program.cs:23]: Why check IsAsciiLetter ?

Comment (Ashish02112001) [reply Program.cs]: Voting they have given for alphabets na. i Have changed to simply IsLetters now.

Comment (asfar-nazar) [reply Program.cs]: Sorry, IsAsciiLetter is the one to be used here.

**Review by asfar-nazar — CHANGES_REQUESTED** [review]
Write tests to validate 'VotesCounter'

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**